Sources/MapboxCoreNavigation/FeedbackEvent.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+/**
+ Feedback event that can be created using `NavigationEventsManager.createFeedback()`.
+ Use `NavigationEventsManager.sendFeedback(_:type:description:)` to send it to the server.
+ Conforms to the `Codable` protocol, so the application can store the event persistently.
+ */
+public class FeedbackEvent: Codable {
+    let coreEvent: CoreFeedbackEvent
+    
+    init(eventDetails: NavigationEventDetails) {
+        let dictionary = try? eventDetails.asDictionary()
+        coreEvent = CoreFeedbackEvent(timestamp: Date(), eventDictionary: dictionary ?? [:])
+    }
+    
+    func update(type: FeedbackType, source: FeedbackSource, description: String?) {
+        coreEvent.eventDictionary["feedbackType"] = type.description
+
+        // if there is a subtype for this event then append the subtype description to our list for this type of feedback
+        if let subtypeDescription = type.subtypeDescription {
+            var subtypeList = [String]()
+            if let existingSubtypeList = coreEvent.eventDictionary["feedbackSubType"] as? [String] {
```

Comment:
Repeated `"feedbackSubType"` string constant, easy to forget to rename the second while renaming the first.

Reply:
Fixed, thanks for noticing.